src/components/Proposal/PredictionBox.tsx

```diff
@@ -158,6 +167,16 @@ export default class PredictionBox extends React.Component<IProps, IState> {
 
     return (
       <div className={wrapperClass}>
+        {showPreStakeModal ?
+          <PreTransactionModal
```

Comment:
Maybe should consider showing a bigger more explicit warning 
message in the modal if already have a sent staking tx in flight (`isStakingPass || isStakingFail`).

Reply:
ill leave that up to Dor

Reply:
Can we tag Dor on this? does he have a github?

Reply:
i created a ticket https://trello.com/c/dtRvcxoI